lib/src/material/material_controls.dart

```diff
@@ -42,7 +42,6 @@ class _MaterialControlsState extends State<MaterialControls>
   late var _subtitlesPosition = Duration.zero;
   bool _subtitleOn = false;
   Timer? _showAfterExpandCollapseTimer;
-  bool _dragging = false;
```

Comment:
why `_dragging` is being removed?

Reply:
same reason as previously, the condition has been changed. Previously it was responsible for hiding icon during manual scroll, but all the standard players I've checked display it and we needed couple of other things that affected the condition.